.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 88
 select = C, E, F, W, B, B950
-extend-ignore = E203, E501, W503
+extend-ignore = E203, E501, E704, W503
```

Comment:
Do we need this? If it's just in one-two places, let's ignore it inline, without changing the config.

Reply:
it's disabled by default and seems to be a fundamentally wrong rule that's incompatible with type annotations, so I don't think it should be enabled.

Reply:
Ah, got it, makes sense, thanks for the explanation 👍